### PR TITLE
Fixes tests with node version 4

### PR DESCRIPTION
**Pull request by rolandpoulter**
* Also cleans up some rather confusing naming with `app.js`

Thanks to @benbp for helping me figure out the fix for this.

### CODE REVIEW

**Comment by rolandpoulter** [comment]
@benbp @heckj @tldavies @DavidjohnBlodgett

File: app.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ var di = require('di');
 
 module.exports = Runner;
 
-di.annotate(Runner, new di.Provide('Http'));
+di.annotate(Runner, new di.Provide('app'));
```
Comment (heckj) [inline app.js]: I love it when you get all semantic pushy!! :-)

**Comment by heckj** [comment]
:+1:

File: app.js (hunk 1 above)
Comment (rolandpoulter) [inline app.js]: I kept thinking to myself "Where is this Http file"

Comment (heckj) [inline app.js]: That was a total Frey-ism

Comment (rolandpoulter) [inline app.js]: Lol.

**Comment by DavidjohnBlodgett** [comment]
:+1:

**Pull request merged**